packages/webapp/src/components/Map/Modals/BulkSensorUploadModal/useValidateBulkSensorData.js

```diff
@@ -54,6 +54,8 @@ export function useValidateBulkSensorData(onUpload, t) {
   const [errorCount, setErrorCount] = useState(0);
   const fileInputRef = useRef(null);
   const [translatedUploadErrors, setTranslatedUploadErrors] = useState([]);
+  const [uploadErrorMessage, setUploadErrorMessage] = useState('');
+  const [errorTypeCode, setErrorTypeCode] = useState(-1);
```

Comment:
Could we maybe map these `errorTypeCode`s to some sort of object of enums, so that other devs can understand what the different codes mean, and it is easier to work with? E.g.
```javascript
const ErrorTypes = {
    EMPTY_FILE: 1,
    INVALID_CSV: 0,
}
```